### PR TITLE
Added a command to wipe the files on the RoboRIO. 

### PR DESCRIPTION
**Pull request by benjiboy50fonz**
As requested, here it is. I think I covered, it all but I haven't been able to test it yet.

### CODE REVIEW

File: pyfrc/mains/cli_undeploy.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        
+        print('SUCCESS: Files have been successfully wiped!')
+        return 0
+        
+    def _delete_dir(self, ssh: sshcontroller.SshController, robot_path: str, robot_filename: str):
+        wipe_dir = PurePosixPath("/home/lvuser")
+        py_wipe_subdir = "py"
+        py_wipe_dir = wipe_dir / py_wipe_subdir
+        
+        sshcmd = "rm -rf %(py_wipe_dir)s"
```
Comment (auscompgeek) [inline pyfrc/mains/cli_undeploy.py:75]: Where is this being interpolated? I can't seem to find it.

Comment (benjiboy50fonz) [reply pyfrc/mains/cli_undeploy.py:78]: That method is being called at line 60 if that's what you're referring to.

File: pyfrc/mains/cli_undeploy.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        try:
+            with sshcontroller.ssh_from_cfg(
+                cfg_filename,
+                username="lvuser",
+                password="",
+                hostname=hostname_or_team,
+                no_resolve=options.no_resolve,
+            ) as ssh:
+
+                if not self._delete_dir(ssh):
```
Comment (virtuald) [inline pyfrc/mains/cli_undeploy.py:61]: This function never returns anything, why are you checking the return value?

Comment (benjiboy50fonz) [reply pyfrc/mains/cli_undeploy.py:61]: Oh good point. I guess there is no reason to return a value. I'll take that out.

File: pyfrc/mains/cli_undeploy.py (hunk 1 above)
Comment (virtuald) [reply pyfrc/mains/cli_undeploy.py:77]: Should be `f"rm -rf {py_wipe_dir}"`

**Comment by virtuald** [comment]
Thanks!

**Pull request merged**